### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mrow4a**
- [x] Needs merge of https://github.com/owncloud/diagnostics/pull/1
- [x] Needs merge of unit tests and travis
- [x] Add code coverage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# OwnCloud Diagnostics
```
Comment (DeepDiver1975) [inline README.md]: ownCloud

**Comment by mrow4a** [comment]
@DeepDiver1975 ready

**Comment by DeepDiver1975** [comment]
coverage is not shown on scrutinizer - see https://scrutinizer-ci.com/docs/tools/external-code-coverage/

**Comment by mrow4a** [comment]
@DeepDiver1975 I am done here, you can preview README here https://github.com/owncloud/diagnostics/blob/b8da23f9e68aeb493a413a1a96bd2f03b7f9672b/README.md

**Pull request merged**